.travis.yml

```diff
@@ -0,0 +1,41 @@
+os: linux
+dist: focal
+vm:
+  size: large
```

Comment:
build and tests for `libevm` should be fairly lightweight, a smaller instance is probably enough :sweat_smile:

Reply:
My bad, I was thinking large size is the smallest:). Just changed it to `medium` = smallest Travis Build VM = 2vCPU and 8 GB of RAM

P.S. The build took 4:30 minutes vs 3:30 which I am assuming is ok?